src/main/java/com/google/sps/servlets/VerifySession.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseAuthException;
+import com.google.firebase.auth.FirebaseToken;
+import com.google.sps.firebase.FirebaseAppManager;
+import javax.servlet.http.Cookie;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/session-verify")
+public class VerifySession extends HttpServlet {
+
+  // Obtain the cookie from the current session
+  static String getCookie(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
Move this to a helper class. Something like `RequestUtils`. This method can also be made `public`.